fastlane/Fastfile

```diff
@@ -33,13 +34,6 @@ def exportEnvVars(env_vars)
         file.puts "#{key}=#{value}"
       end
     end
-  else
-    puts "Saving environment variables in parent shell..."
-    env_vars.each do |key, value|
-      puts "#{key}=#{value}"
-      command = "export #{key}=#{value}"
-      system(command)
```

Comment:
Interestingly, this never worked. I only added it so that it might be possible to run these commands locally, but never really tested it. Ruby is spawned in a subprocess, so there's no way for it to directly modify the environment of the parent shell. It can update a file or output something directly, but those outputs need to be processed by the parent shell to be added as environment variables.

So what was happening is that this was working to set environment variables in one step of a job and then read them in another, because they were being set in the GITHUB_ENV file, and that file is loaded into the environment at the start of each step. But trying to access those variables in the _same step_ wouldn't work.

The fix to this was nice and simple - just add a `source "$GITHUB_ENV"` in the same step before trying to read environment variables.